cmd/endorse.go

```diff
@@ -120,15 +120,24 @@ func validateSnpFlags(f *sev.SnpEndorsementRequest) error {
 // The SVN for SEV, TDX, and non-CoCo are all the same for now.
 func scrtmMain(path string) (bool, uint32, error) {
 	// Read the bundled SVN if it exists.
-	versionPath := strings.Replace(path, ".fd", "_scrtm_ver.pb", 1)
-	if versionBytes, err := os.ReadFile(versionPath); err == nil {
-		var version edk2pb.SCRTMVersion
-		if err := proto.Unmarshal(versionBytes, &version); err != nil {
-			return false, 0, err
+	var versionBytes []byte
+	var err error
+	for _, path := range []string{
+		strings.Replace(path, ".fd", "_scrtm_ver.pb", 1),
```

Comment:
nit: consider adding a test for all 4 cases of these paths existing or not

Reply:
Rather not. This is minor glue to remove the _scrtm_ver.pb file after the next release.